code/modules/admin/fun_verbs.dm

```diff
@@ -1001,27 +1001,40 @@
 	message_admins("[ADMIN_TPMONTY(IF)] started being imaginary friend of [ADMIN_TPMONTY(L)].")
 
 
-	
 /datum/admins/proc/force_dropship()
 	set category = "Fun"
 	set name = "Force Dropship"
 
 	if(!check_rights(R_FUN))
 		return
 
-	if(!length(SSshuttle.dropships))
+	if(!length(SSshuttle.dropships) && !SSshuttle.canterbury)
+		message_admins("no ships")
```

Comment:
I assume this is temporary?

Reply:
yeah this whole thing is borked and needs work